project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.6.2
+sbt.version=1.5.8
```

Comment:
I don't think we need to downgrade sbt version.

Reply:
I've now reverted the version back to 1.6.2 in the latest commit